Classes/ViewHelpers/Uri/AjaxActionViewHelper.php

```diff
@@ -29,6 +29,9 @@
  * index.php?id=123&tx_typoscriptrendering[context]={"record":"tt_content_123","path":"tt_content.list.20.myextension_plugin"}&tx_myextension_plugin[action]=show&tx_myextension_plugin[controller]=Standard&cHash=xyz
  * (depending on the current page and your TS configuration)
  * </output>
+ *
+ * Please note that this only works for plugins that are using the PLUGIN_TYPE_PLUGIN type,
+ * not for PLUGIN_TYPE_CONTENT_ELEMENT plugins.
```

Comment:
These are constants from the \TYPO3\CMS\Extbase\Utility\ExtensionUtility class. Both constants are valid values for the argument $pluginType of \TYPO3\CMS\Extbase\Utility\ExtensionUtility::configurePlugin:
https://github.com/TYPO3/TYPO3.CMS/blob/master/typo3/sysext/extbase/Classes/Utility/ExtensionUtility.php#L45

(The default value is the one that works.)